src/server/components/proxyMiddleware/controllers.ts

```diff
@@ -1,28 +1,70 @@
 import type { Request, Response } from "express";
 import { validateCountryLower } from "server/models/listItem/providers/helpers";
 import { listExists } from "server/components/proxyMiddleware/helpers";
+import { countriesList } from "server/services/metadata";
+import * as Routes from "./routes";
+import { camelCase } from "lodash";
 
-export async function lawyersPostController(req: Request, res: Response) {
+export function getStartPageController(req: Request, res: Response) {
+  const routeUrl = req.path;
+  const serviceType = routeUrl.split("/")[2];
+  const countrySelectFullUrl = Routes[camelCase(serviceType) as "lawyers" | "funeralDirectors"].countrySelect;
```

Comment:
I don't think this is necessary - see my comments on router

but this is better achieved by `req.params.serviceType` 

```
 res.render(`apply/${serviceType}/start`) // no need to provide a path to context
```


in any start.njk, 

```
  {{ govukButton({
    text: "Start now",
    href: "which-country-list-do-you-want-to-be-added-to",
    isStartButton: true,
    element: "a"
  }) }}
```

 clicking this link will take you to 

`/application/${serviceType}/which-country...`

without a leading slash - the link will be relative to the subdirectory or path you're in. 

e.g. the current url would be `application/${serviceType}/start`, so the subdirectory would be `application/${serviceType}` + the href.